app/views/cases/respond.html.slim

```diff
@@ -1,8 +1,13 @@
+= GovukElementsErrorsHelper.error_summary @case.object, @case.error_summary_message, ''
+
 - content_for :page_title do
   = t('page_title.mark_as_sent_page', case_number: @case.number)
 
 - content_for :heading
   = t('.heading')
+  br
```

Comment:
Having these two in the heading looks a little bit odd to me:

![screenshot_2018-08-03 marking response as sent - 180803001 - track-a-query](https://user-images.githubusercontent.com/15608/43645412-b7a6f394-9729-11e8-968f-5f0ece048b10.png)

I don't see designs for it but it seems to me that other pages don't have the page heading and the case subject both so big.

Reply:
@misaka I agree with you. Its something I've been speaking to Mia about, oddly its on this page and some of the "Close cases" page.

Reply:
Here are the designs: https://e25x69.axshare.com/#g=1&p=ico_appeal_case_sent_to_ico
If we want to change this I suggest doing it in another PR once you've agreed a change with Mia

Reply:
@beccapearce Thanks for the link. As discussed in stand-up, let's just get this through and address this separately, if at all.